package/kernel/mac80211/patches/ath/558-ath9k-only-mask-use_eeprom-on-of-noeeprom.patch

```diff
@@ -0,0 +1,19 @@
+--- a/drivers/net/wireless/ath/ath9k/init.c
++++ b/drivers/net/wireless/ath/ath9k/init.c
+@@ -636,14 +636,15 @@ static int ath9k_of_init(struct ath_softc *sc)
+ 		ret = ath9k_eeprom_request(sc, eeprom_name);
+ 		if (ret)
+ 			return ret;
++
```

Comment:
extra empty line.

Reply:
I had because the code block becomes:
```
               if (ret)                                                                                      
                       return ret;                                                                           
                                                                                                            
               ah->ah_flags &= ~AH_USE_EEPROM;  
```
which it seem is common to have based on a few lines lower:
```
       mac = of_get_mac_address(np);                                                                         
       if (mac)                                                                                              
               ether_addr_copy(common->macaddr, mac);                                                        
                                                                                                             
       ah->ah_flags |= AH_NO_EEP_SWAP; 
```
Is there an official style guide thing on this, or was the latter not quite right, but made it in?

Reply:
I'm not sure about an official style guide
my personal preference is to keep the blank line as it separates one code-block (for validating the return code) from a code-block with a completely different purpose (setting some flag)

Reply:
No, sorry. Please disregard my comment. I'm mistaken here and it looks right the way it is.